css/styles.css

```diff
@@ -79,14 +133,171 @@ li {
   font-size: 16px;
   line-height: 24px;
   color: #7f8cff;
-  margin: 0;
+  margin: 0 24px;
 }
 
 .icons-media {
   display: flex;
-  justify-content: left;
 }
 
 .icon {
   padding-right: 20px;
+  padding-left: 0;
+  float: left;
+  text-align: left;
+}
+
+.counter {
+  padding-right: 10px;
+  float: left;
+  text-align: left;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.canopy {
+  padding-right: 10px;
+  float: left;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.language {
+  float: left;
+  text-align: left;
+  background: #ebebff;
+  border-radius: 8px;
+  margin-right: 8px;
+  margin-bottom: 10px;
+  padding: 4px 12px;
+  border: #ebebff;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+}
+
+.project {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  background: #fff;
+  border: 1px solid  #6070ff;
+  box-sizing: border-box;
+  border-radius: 16px;
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+}
+
+.button_project {
+  width: 295px;
+  height: 72px;
+  top: 260px;
+  padding-left: 16px;
+  border: none;
+}
```

Comment:
Kindly add the button interactions from the Figma template (hover, active ) to the 'see project' button 😃✔😃

![image](https://user-images.githubusercontent.com/87448628/164761409-7d232f09-ba05-4ce6-8e14-a1c0fc725494.png)